frontend/src/app/shared/Restaurant.ts

```diff
@@ -5,4 +5,6 @@ export interface Restaurant {
   lng: number;
   photos: string[];
   photo: string;
+  routeDuration: string;
+  routeDistance: string;
 }
```

Comment:
in the component template you are using `{{restaurant.route.distance}}`  to display the distance .
so you need to respect this here .
you can create new Interface Route with 2 properties ( distance, duration) ; and add a route object to restaurant .
or change   `{{restaurant.route.distance}}` to  `{{restaurant.routeDistance}}`